README.md

```
@@ -36,33 +44,86 @@ pip install -r requirements.txt
-# Install dependencies (using pnpm)
+# Install dependencies
 pnpm install
 
-# Start the development server
+# Start development server
 pnpm dev
 ```
 
-### 4. Access the Application
+#### 5. Access the Application
 - **Frontend**: http://localhost:3000
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)